bundles/com.espressif.idf.core/plugin.xml

```diff
@@ -260,4 +260,22 @@ config-only component and an interface library is created instead."
            name="KCONFIG_PROJBUILD">
      </property>
   </extension>
+  <extension
+        point="org.eclipse.core.variables.dynamicVariables">
+     <variable
+           description="%openocd_bin_path"
+           name="openocd_path"
+           resolver="com.espressif.idf.core.variable.OpenocdVariableResolver">
+     </variable>
+     <variable
+           description="%openocd_exe"
+           name="openocd_exe"
+           resolver="com.espressif.idf.core.variable.OpenocdVariableResolver">
+     </variable>
+     <variable
+           description="%openocd_scripts"
+           name="OPENOCD_SCRIPTS"
+           resolver="com.espressif.idf.core.DynamicVariableResolver5">
```

Comment:
DynamicVariableResolver5 ? I couldn't find relevant class for this?

Reply:
Hi @kolipakakondal, thanks for noticing. Looks like Eclipse didn't pick up this class after Refactoring -> Rename. Fixed in the latest commit